src/navigator/Navigator.tsx

```diff
@@ -711,7 +730,14 @@ function RootStackScreen() {
     <RootStack.Navigator
       screenOptions={{ snapPoints: ['100%'], backdropComponent: renderBackdrop }}
     >
-      <RootStack.Screen name={Screens.MainModal} component={ModalStackScreen} />
+      {isSanctionedCountry ? (
+        <RootStack.Screen
```

Comment:
this feels kinda hacky 😅 weird to have `nativeBottomSheets(RootStack)` inside the sanctioned screens root stack.

i'd suggest to move the implementation inside `NavigatorWrapper`, and handling it like `Screens.BackupForceScreen` in the useEffect in line 84 :)

Reply:
I also thought that it is hacky. Using useEffect is a lot better approach! Thanks a lot!
I changed it as you said

Reply:
I just found a problem with a `useEffect` implementation.
Users can bypass it by sliding back...

Please see this video.
https://user-images.githubusercontent.com/47931506/185673311-001c05e0-b07e-496c-b454-828b0ad3acc9.mp4